TeamSnapSDK/SDK/DataTypes/TSDKTeam.h

```diff
@@ -146,7 +146,7 @@
 - (void)setTimeZone:(NSTimeZone *_Nonnull)timeZone;
 - (NSTimeZone *_Nullable)timeZone;
 
-- (instancetype _Nullable)initWithName:(NSString *_Nonnull)name locationCountry:(NSString *_Nonnull)locationCountry locationPostalCode:(NSString *_Nullable)locationPostalCode ianaTimeZoneName:(NSString *_Nullable)ianaTimeZoneName sportId:(NSString *_Nonnull)sportId;
+- (instancetype _Nullable)initWithName:(NSString *_Nonnull)name locationCountry:(NSString *_Nonnull)locationCountry locationPostalCode:(NSString *_Nullable)locationPostalCode ianaTimeZoneName:(NSString *_Nullable)ianaTimeZoneName sportId:(NSString *_Nonnull)sportId createTeamPlan:(NSString * _Nonnull)createTeamPlan;
```

Comment:
For backwards compatibility, would it be better to create a second method definition (keep the one that does NOT have `createTeamPlan` and add this new one that DOES)... and have the old method call the new w/ `true` as the default?